examples/src/main/java/io/dapr/examples/bindings/README.md

```diff
@@ -0,0 +1,130 @@
+
+Before getting into the application code, follow these steps in order to setup a local instance of Kafka. This is needed for the local instances.  Steps are:
+
+1. Run `docker-compose -f ./docker-compose-single-kafka.yml up -d` to run the container locally
+2. Run `docker ps` to see the container running locally: 
+
+```bash
+342d3522ca14        kafka-docker_kafka                      "start-kafka.sh"         14 hours ago        Up About
+a minute   0.0.0.0:9092->9092/tcp                               kafka-docker_kafka_1
+0cd69dbe5e65        wurstmeister/zookeeper                  "/bin/sh -c '/usr/sb…"   8 days ago          Up About
+a minute   22/tcp, 2888/tcp, 3888/tcp, 0.0.0.0:2181->2181/tcp   kafka-docker_zookeeper_1
+```
+Click [here](https://github.com/wurstmeister/kafka-docker) for more information about the kafka broker server.
+
+### Running the Input binding sample
+
+The input binding sample uses the Spring Boot´s DaprApplication class for initializing the `InputBindingController`. In `InputBindingExample.java` file, you will find the `InputBindingExample` class and the `main` method. See the code snippet below:
+
+```java
+public class InputBindingExample {
+
+  public static void main(String[] args) throws Exception {
+    ///..
+    // If port string is not valid, it will throw an exception.
+    int port = Integer.parseInt(cmd.getOptionValue("port"));
+
+    // Start Dapr's callback endpoint.
+    DaprApplication.start(port);
+  }
+///...
+}
+```
+
+The `InputBindingController` is a Spring REST controller that handles the input binding request as a POST. The Dapr's sidecar is the one that performs the actual call to the controller, based on the binding features and the output binding action. 
+
+ Execute the follow script in order to run the Input Binding example:
+```sh
+cd to [repo-root]/examples
+dapr run --app-id inputbinding --app-port 3000 --port 3005 -- mvn exec:java -D exec.mainClass=io.dapr.examples.bindings.InputBindingExample -D exec.args="-p 3000"
+```
+
+### Running the Output binding sample
+
+The output binding application is a simple java class with a main method that uses the Dapr Client to invoke binding.
+
+In the `OutputBindingExample.java` file, you will find the `OutputBindingExample` class, containing the main method. The main method declares a Dapr Client using the `DaprClientBuilder` class. The client publishes events using `invokeBinding` method. See the code snippet below:  
+```java
+public class OutputBindingExample {
+///...
+  public static void main(String[] args) throws Exception {
+    DaprClient client = new DaprClientBuilder().build();
+    final String BINDING_NAME = "bindingSample";
+    ///...
+    MyClass myClass = new MyClass();
+    myClass.message = "hello";
+
+    System.out.println("sending first message");
+    client.invokeBinding(BINDING_NAME, myClass); //Binding a data object
+    ///..
+    final String m = "cat";
+    System.out.println("sending " + m);
```

Comment:
First message and second message are handled differently. Handle those the same way, please.